Makefile

```diff
@@ -69,7 +69,11 @@ cri-containerd: check-gopath
 	   $(PROJECT)/cmd/cri-containerd
 
 test:
-	go test -timeout=1m -v -race ./pkg/... $(BUILD_TAGS)
+ifeq ($(TRAVIS),true)
+	go test -short -timeout=1m -v -race ./pkg/... $(BUILD_TAGS)
```

Comment:
it's an option to allow for skipping some unit tests in travis..  

The alternative is to configure travis to run containerd, runc, etc. Or to delete all unit tests that require containerd, runc, etc.